src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -1123,11 +1123,11 @@ def delete_policy(cmd, client, resource_group_name, vault_name,
 
 
 # region KeyVault Key
-def create_key(client, key_name=None, protection=None,  # pylint: disable=unused-argument
```

Comment:
The argument registrations of `key_name` and `name` seem duplicated:

For `key_name`:

https://github.com/Azure/azure-cli/blob/2af2d4f78d9d5867d7d889c0a47fe0da21b944b7/src/azure-cli/azure/cli/command_modules/keyvault/_params.py#L276-L279

For `name`:

https://github.com/Azure/azure-cli/blob/2af2d4f78d9d5867d7d889c0a47fe0da21b944b7/src/azure-cli/azure/cli/command_modules/keyvault/_params.py#L452-L456

I think we should remove one.

Reply:
Exactly. That's what I did in `_params.py`. I have removed the track1 definition `key_name`.